x-pack/plugins/infra/public/components/asset_details/hooks/use_asset_details_state.ts

```diff
@@ -10,15 +10,12 @@ import type { AssetDetailsProps } from '../types';
 import { useMetadataStateProviderContext } from './use_metadata_state';
 
 export interface UseAssetDetailsStateProps {
-  state: Pick<
-    AssetDetailsProps,
-    'asset' | 'assetType' | 'overrides' | 'onTabsStateChange' | 'renderMode'
-  >;
+  state: Pick<AssetDetailsProps, 'asset' | 'assetType' | 'overrides' | 'renderMode'>;
 }
 
 export function useAssetDetailsState({ state }: UseAssetDetailsStateProps) {
```

Comment:
I'm under the impression that the names here have become confusing with the introduction of the `use_asset_details_url_state`. I wonder if it should be called something along the lines of `useAssetDetailsRenderInfo`.

Reply:
makes sense, will rename it 👍